internal/controller/apiextensions/composite/composed.go

```diff
@@ -359,6 +350,17 @@ func (r *APIDryRunRenderer) Render(ctx context.Context, cp resource.Composite, c
 		}
 	}
 
+	// Fix(2416): composed labels and annotations should be rendered after patches are applied
```

Comment:
This `Fix(NNNN)` practice is new to me. Do you think it's something we should adopt? Typically I've mostly trusted git to create the audit trail we need through commit messages, git blame, etc such that code can be traced back to a PR. We do specifically link to a PR or via its full URL in some cases when we feel that the reader would benefit from additional context.

Reply:
The idea is to use a concise format similar to GitHub's [autolinked references](https://docs.github.com/en/github/writing-on-github/working-with-advanced-formatting/autolinked-references-and-urls) to refer to a related issue (in case of a `Fix`, to refer to an issue reporting a bug) in order to add some more context on the source listing itself. I think it draws attention to the specific site and warns future maintainers about a potential pitfall on the site. But we can of course embed full URLs or just leave comments (+ git blame) to achieve the same goal.